library/query_label_context_injection.yml

```diff
@@ -0,0 +1,28 @@
+id: RP-5
+
+info:
+  name: Label Context Injection
+  severity: high
+  description: Label Injection is caused by using label variables in inline scripts
+  full-description: |
+    Creating a new pull request could be submitted with a label that contains special characters.
+    By using a label variable in an inline script, an attacker can inject arbitrary code into the build process.
+  references:
+    - https://docs.github.com/en/actions/security-guides/security-hardening-for-github-actions
+  tags:
+    - injection
+    - unauthenticated
```

Comment:
To my understanding labels requires certain permissions.
I think Triage ones - https://docs.github.com/en/organizations/managing-user-access-to-your-organizations-repositories/managing-repository-roles/repository-roles-for-an-organization
So maybe we need to alter severity description, and remove unauthenticated tag